riff-raff.yaml

```diff
@@ -19,3 +19,6 @@ deployments:
         AmigoStage: PROD
         BuiltBy: amigo
       amiEncrypted: true
+      templatePath: Amiable.template.json
+      cloudFormationStackName: amiable-gucdk-
```

Comment:
I think we can allow Riff-Raff to discover the stack based on tags (and omit `cloudFormationStackName` and `appendStageToCloudFormationStackName`)? I think this is what we're doing for [Prism](https://github.com/guardian/prism/blob/47f1630a266236c155b9294626973fbb3088df2d/riff-raff.yaml#L4-L14).

Reply:
Ah! That's nicer! Will do this in another PR.

Reply:
I think Riff-Raff is already doing this, which is more reason to remove this property as it appears to be entirely unused.

![image](https://user-images.githubusercontent.com/836140/129029221-f449cd66-3e6c-48c8-8df1-ee224010f679.png)

Reply:
https://github.com/guardian/amiable/pull/77